Heroes Handbook, Main File

```diff
@@ -1257,17 +1257,15 @@ Tauren share certain characteristics no matter what tribe they come from.
 
 ***Speed.*** Your base walking speed is 30 feet.
 
-***Powerful Build.*** You count as one size larger when determining your carrying capacity and the weight you can push, drag, or lift.
-
-\columnbreak
+***Gore.*** Your horns are natural weapons, which you can use to make an unarmed strike. If you hit with them, you deal 1d8 + your Strength modifier piercing damage.
```

Comment:
```suggestion
***Gore.*** Your horns are natural weapons, which you can use to make an unarmed strike. When you hit with them, the target takes piercing damage equal to 1d6 + your Strength modifier```

Reply:
Off of the stats for Minotaurs out of Unearthed Arcana, the damage die on using their horns is a 1d6 rather than a 1d8. This would also put it on par with a monk's Unarmed Strike all the way up until 11th level, where Unarmed Strike takes over as the more powerful attack.